src/main/java/io/gatling/mojo/AbstractGatlingMojo.java

```diff
@@ -50,12 +66,37 @@
   @Parameter(defaultValue = "${session}", readonly = true)
   protected MavenSession session;
 
+  /** List of include patterns to use for scanning. Includes all simulations by default. */
+  @Parameter(property = "gatling.includes")
+  protected String[] includes;
+
+  /** List of exclude patterns to use for scanning. Excludes none by default. */
+  @Parameter(property = "gatling.excludes")
+  protected String[] excludes;
```

Comment:
What's the meaning of includes and excludes for children mojos such as `RecorderMojo`?

Reply:
I was trying to share the simulation class discovery logic with `EnterpriseStartMojo` and also to have less duplicated code between the different mojos, but this doesn't really work, true (not just `RecorderMojo`, but also `EnterprisePackageMojo` and `EnterpriseUploadMojo` don't use it either)...

However, on second thoughts, simulation class discovery for `EnterpriseStartMojo` should probably _not_ use `includes`/`excludes` filters either and just use the full list of simulation classes discovered. In the future interactive mode, the user will be able to choose one from the list; and I think the filter system only really makes sense when running several simulations in one command, which is possible when running locally but not when creating a simulation on Gatling Enterprise.

I'm reworking this.